apps/syntrest/src/main/java/no/nav/registre/syntrest/controllers/SyntController.java

```diff
@@ -163,17 +141,12 @@
             @RequestParam int numToGenerate,
             @ApiParam(value = "Verdi som vil overskrive alle ArbeidetTimerSum i meldekort")
             @RequestParam(required = false) Double arbeidstimer
-    ) {
+    ) throws ApiException, InterruptedException {
         InputValidator.validateInput(numToGenerate);
         InputValidator.validateInput(InputValidator.INPUT_STRING_TYPE.MELDEGRUPPE, meldegruppe);
-        String url = Objects.isNull(arbeidstimer)
-                ? arenaMeldekortUrl
-                : arenaMeldekortUrl + "?arbeidstimer=" + arbeidstimer;
```

Comment:
Trenger fremdeles å kunne legge til arbeidstimer parameter hvis den ikke er null ;)

Reply:
La til funksjonalitet for queryParameters i SyntConsumer. Akkurat den logikken begyyyynner å lukte at den burde vært en URI klasse i stedet for String i SyntConsumer, men gjør ikke noe med det helt enda.

Reply:
Skjønner 👍